core/shared/src/main/scala-2.11-2.12/ChunkBuilder.scala

```diff
@@ -43,7 +43,7 @@ private[zio] object ChunkBuilder {
   /**
    * Constructs a generic `ChunkBuilder`.
    */
-  def make[A](): ChunkBuilder[A] =
+  def make[A]: ChunkBuilder[A] =
```

Comment:
We should keep the parens. It is a Scala convention for side effecting nullary methods to use parens to distinguish them from vals at the call-site.

Reply:
I was thinking so as well, but noticed that the standard library doesn't use parens in for example ArrayBuilder#make.
Discussion why they decided so is here:
https://github.com/scala/collection-strawman/issues/520

Should we still stick with parens?

Reply:
I think it was a mistake to not use parens in `ArrayBuilder.make` and that they would change that decision now.

Reply:
alright, updated